### ISSUE TITLE
Poor Average Precision values reported for baseline training of sim10k

### ISSUE THREAD

**Issue by Vigneshwar73** [justinkay/aldi#65]
Hi, I am trying to implement your default implementation of sim10k -> CS. As the first step, I am trying to conduct the baseline training using R-CNN and FPN. However, the reported evaluation accuracy is very poor compared to the values shown in your paper. Could you help me with this issue?

[log.txt](https://github.com/user-attachments/files/19759024/log.txt)
[metrics.json](https://github.com/user-attachments/files/19759025/metrics.json)
[config.txt](https://github.com/user-attachments/files/19759089/config.txt)

**Comment by justinkay** [comment]
Hi @Vigneshwar73 , it would be helpful to know all the things you changed from the default config. At first glance, it looks like the batch size was changed, but the number of iterations was not, meaning you are not training the full amount. If this is correct, try either reverting to the default batch size or training longer and see how that goes.

**Comment by Vigneshwar73** [comment]
Hi @justinkay , Thanks for the prompt response. I had changed the learning rate to 0.001 as initially my training diverged. So I thought lower learning rate might help and also I had changed the batch size to compensate for my GPU capability. If I understand correctly, iterations required for one complete training of the whole dataset should be 10000/48 =208.33. So, even if I reduce batchsize iterations for one complete training of dataset should be 10000/16 = 625. This means I am training the whole dataset right? 

Anyways, I will try your suggestion and train for longer.

**Comment by justinkay** [comment]
@Vigneshwar73 closing for now since I have not heard from you, let me know if there are any further issues.
